server/routes/incidents_routes.py

```
@@ -2027,6 +2027,47 @@ def get_recent_unlinked_incidents(user_id):
+                cursor.execute(
+                    """SELECT r.id, r.action_id, r.status, r.chat_session_id,
+                              r.started_at, r.completed_at, r.error,
+                              a.name AS action_name
+                       FROM action_runs r
+                       JOIN actions a ON a.id = r.action_id
+                       WHERE r.incident_id = %s AND r.org_id = %s
+                       ORDER BY r.started_at DESC""",
+                    (incident_id, org_id),
+                )
+                cols = [d[0] for d in cursor.description]
+                rows = [dict(zip(cols, row)) for row in cursor.fetchall()]
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)